src/main/java/de/rabea/logger/ConsoleLogger.java

```diff
@@ -0,0 +1,22 @@
+package de.rabea.logger;
+
+import de.rabea.Console;
+
+public class ConsoleLogger implements Logger {
+
+    private Console console;
+
+    public ConsoleLogger(Console console) {
+        this.console = console;
+    }
+
+    @Override
+    public void log(String message) {
+        console.write(message);
+    }
+
+    @Override
+    public String getLogs() {
+        return "";
```

Comment:
@felipesere I had to implement this method as part of the Interface to return the logs - but it only works with the FileLogger. So I'm just returning an empty string here but I feel that that's not ideal. Do you have any other suggestions how to resolve this?